lib/src/rules/always_specify_types.dart

```diff
@@ -102,6 +104,23 @@ class AlwaysSpecifyTypes extends LintRule {
 }
 
 class _Visitor extends SimpleAstVisitor<void> {
+  static const LintCode keywordCouldBeTypeCode = LintCode(
+      "always_specify_types", // ignore: prefer_single_quotes
+      "Specify the type annotation.", // ignore: prefer_single_quotes
+      correction: "Try replacing '{0}' with '{1}'.");
+
+  static const LintCode keywordCouldBeSplitToTypesCode = LintCode(
+      "always_specify_types", // ignore: prefer_single_quotes
+      "Specify type annotations.", // ignore: prefer_single_quotes
+      correction:
+          "Try splitting the declaration and specify the different type annotations."); // ignore: prefer_single_quotes
+
+  static const LintCode specifyTypeCode = LintCode(
+      "always_specify_types", // ignore: prefer_single_quotes
+      "Specify the type annotation.", // ignore: prefer_single_quotes
+      correction:
+          "Try specifying the type '{0}'."); // ignore: prefer_single_quotes
```

Comment:
If you're making other changes, then I'll note that this ignore isn't needed. If you're not making other changes then you can leave it.

Similarly, if you want to, it's perfectly fine to use single quotes for the name of the code (on lines 108, 113, and 119). Your choice.

Reply:
I understood from https://github.com/dart-lang/linter/pull/2863#discussion_r684798583, that all `LintCode` usage should have double quotes.

Reply:
Then I wasn't very clear. The convention we follow is that messages (the problem message and the correction message in this case) are always in double quotes so that they're consistent and allow for having single quotes in the messages themselves. The code name is usually in single quotes because it isn't a message and hence gets treated like any other string.

Reply:
nit: The ignore on line 110 shouldn't be needed because the string contains single quotes.